src/sass/components/_signup.scss

```
@@ -0,0 +1,33 @@
+.input-item {
```

```suggestion
.input-group {
```
Đặt tên có ý nghĩa hơn nhé